reference/docs-conceptual/components/vscode/using-vscode.md

```diff
@@ -204,12 +208,13 @@ debugging.
 
 ### Workspace debugging
 
-Workspace debugging refers to debugging in the context of a folder that you have opened in Visual Studio Code using **Open Folder...** from the **File** menu.
-The folder you open is typically your PowerShell project folder and/or the root of your Git repository.
+Workspace debugging refers to debugging in the context of a folder that you have opened in Visual
+Studio Code using **Open Folder...** from the **File** menu. The folder you open is typically your
+PowerShell project folder and/or the root of your Git repository.
 
-Even in this mode, you can start debugging the currently selected PowerShell script by simply pressing F5.
-However, workspace debugging allows you to define multiple debug configurations other than just debugging the currently open file.
-For instance, you can add a configurations to:
+Even in this mode, you can start debugging the currently selected PowerShell script by simply
+pressing F5. However, workspace debugging allows you to define multiple debug configurations other
```

Comment:
Should F5 be in codefence as a key click (like R above)?  Also, instead of code fence, should those be in kbd blocks?

Reply:
Fixed